pywarp10/gts.py

```diff
@@ -31,7 +31,7 @@ def is_gts(x: Any) -> bool:
     Returns:
         True if x is a GTS.
     """
-    if not isinstance(x, Dict):
+    if not isinstance(x, Dict) or len(x) == 0:
```

Comment:
` x=0
>>> len(x)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'int' has no len()`

int doesn't have len()

Reply:
Yes but int is not a Dict either so it won't pass the first condition.

```
>>> pywarp10.gts.is_gts(1)
False
```